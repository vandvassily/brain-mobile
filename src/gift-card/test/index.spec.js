import GiftCard from '..';
import { mount } from '../../../test';

test('create a GiftCard', () => {
  const wrapper = mount(GiftCard);

  expect(wrapper).toMatchSnapshot();
});
