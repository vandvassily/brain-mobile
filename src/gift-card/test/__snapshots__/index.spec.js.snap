// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`create a GiftCard 1`] = `
<div class="gift-item"><img src="" alt="" class="gift-item-img">
  <div class="gift-content"><span class="gift-title">【赠品】&nbsp;</span> <span class="gift-nums">X1</span></div>
</div>
`;
